messages/run-dfa.md

```diff
@@ -1,67 +1,50 @@
 
 # flags.ruledisablewarningviolationSummary
 
-disable warning violations from Salesforce Graph Engine. Alternatively, set value using environment variable `SFGE_RULE_DISABLE_WARNING_VIOLATION`
+Disable warning violations from Salesforce Graph Engine. Inherits its value from the "SFGE_RULE_DISABLE_WARNING_VIOLATION" environment variable, if set.
 
 # flags.ruledisablewarningviolationDescription
 
-Disables warning violations, such as those on StripInaccessible READ access, to get only high-severity violations (default: false). Inherits value from SFGE_RULE_DISABLE_WARNING_VIOLATION env-var if set.
+Examples of warning violations include those on StripInaccessible READ access, to get only high-severity violations (default: false).
 
 # flags.rulethreadcountSummary
 
-specify number of threads that evaluate DFA rules. Alternatively, set value using environment variable `SFGE_RULE_THREAD_COUNT`. Default is 4
-
-# flags.rulethreadcountDescription
-
-Specifies the number of rule-evaluation threads or how many entry points can be evaluated concurrently. Inherits its value from the SFGE_RULE_THREAD_COUNT environment variable, if set. The default is 4.
+Number of DFA rule-evaluation threads or how many entry points can be evaluated concurrently. Inherits its value from the "SFGE_RULE_THREAD_COUNT" environment variable, if set.
 
 # flags.rulethreadtimeoutSummary
 
-specify timeout for individual rule threads in milliseconds. Alternatively, set the timeout value using environment variable `SFGE_RULE_THREAD_TIMEOUT`. Default: 900000 ms
-
-# flags.rulethreadtimeoutDescription
-
-Specifies the time limit for evaluating a single entry point in milliseconds. Inherits its value from the SFGE_RULE_THREAD_TIMEOUT environment variable, if set. The default is 900,000 ms or 15 minutes.
+Time limt, in milliseconds, for evaluating a single entry point for individual rule threads. Inherits its value from the "SFGE_RULE_THREAD_TIMEOUT" environment variable, if set.
 
 # flags.sfgejvmargsSummary
 
-specify Java Virtual Machine (JVM) arguments to optimize Salesforce Graph Engine execution to your system (optional)
-
-# flags.sfgejvmargsDescription
-
-Specifies Java Virtual Machine arguments to override system defaults while executing Salesforce Graph Engine. For multiple arguments, add them to the same string separated by space.
+Java Virtual Machine (JVM) arguments to override system defaults while executing Salesforce Graph Engine; separate multiple arguments by a space.
 
 # flags.targetSummary
 
-source code location
+Source code location.
 
 # flags.targetDescription
 
-Specifies the source code location. Use glob patterns or specify individual methods with #-syntax. Multiple values are specified as a comma-separated list. Default is ".".
+Use glob patterns or specify individual methods with #-syntax. Multiple values are specified as a comma-separated list.
```

Comment:
@jshackell-sfdc Curious: what's the purpose behind removing the default values from the doc descriptions? cc: @stephen-carter-at-sf

Reply:
Default flag values are just another property, along with whether it's required, its data type, and a lot more.  So to me it makes sense to define these properties in the code rather than in the documentation.  The `--help` output will then show the default value in the same way as other core commands.